docs/en/operation/Metrics-System.md

```diff
@@ -71,6 +76,9 @@ If Alluxio is deployed in a cluster, this file needs to be distributed to all th
 Then, start Alluxio, CSV files containing metrics will be found in the `sink.csv.directory`. The
 file name will correspond with the metric name.
 
+To configure another sink, simply update `sink.csv.class` to sink of choice. Sink specific parameters can be found 
```

Comment:
I actually think that the property name changes depending on which sink is used. i.e. the JMX sink. https://github.com/Alluxio/alluxio/blob/master/conf/metrics.properties.template#L70

I think there are some instructions earlier which say a user can use the examples in the `metrics.properties.template`. I think that's probably enough. Maybe just another reminder sentence here to tell people where to look for more configurations.

Reply:
got it, updated!